docs/uperf.md

```diff
@@ -108,7 +111,7 @@ size.
 For example:
 ```yaml
       samples: 3
-      pair: 1
```

Comment:
can we have an example of pairs and an example of density?

Reply:
ok. Will add. 
TL;TR we support both 'pair' and 'density' notations with 'pair' of higher precedence. If 'pair' is defined, it will use 'pair', and the behavior is backward compatible i.e 'pair' specifies number of client-server pairs. In the absence of  'pair', it will use 'density_range', and 'density_range' supports iteration. For example denstity_range: [4,4] is the same as pair: 4.  density_range: [1,4] will iterate 1 pair, 2 pair, 3 and 4.